back/engines/commercial/multi_tenancy/lib/tasks/core/clean_up_image_records.rake

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+namespace :cl2back do
+  desc 'Remove image records not associated with resource or with nil value for image field'
+  # Usage:
+  # Dry run (no changes): cl2back:clean_up_image_records
+  # Execute (destroys records!): cl2back:clean_up_image_records EXECUTE_CLEAN_UP_UNUSED_IMAGE_RECORDS=true
+  # We use an ENV var as this seems to work better with specs.
+  task clean_up_unused_image_records: :environment do
+    dry_run = true unless ENV.fetch('EXECUTE_CLEAN_UP_UNUSED_IMAGE_RECORDS') == 'true'
```

Comment:
I tried using a regular rake task argument here, but I wasn't able to consistently set it in the related spec. I tried various versions of using the `.invoke` command to pass the argument to the task in the spec (as described [here](https://stackoverflow.com/questions/47390842/how-to-test-rake-task-when-arguments-are-passed)), but could never get both tests to run successfully, only the first test of two.

I made the ENV var name very specific to avoid colliding with any other ENV that might be in use.

Reply:
Did you try calling `execute` with options? https://stackoverflow.com/a/8704965

Reply:
> Did you try calling execute with options? https://stackoverflow.com/a/8704965

There's certainly some variations in that post I have not tried. Thanks.

Reply:
Yup, that works. :+1: